contracts/governance/Governed.sol

```diff
@@ -51,15 +51,16 @@ contract Governed {
      * This function must called by the pending governor.
      */
     function acceptOwnership() external {
+        address _pendingGovernor = pendingGovernor;
```

Comment:
I think maybe this line could be `address oldPendingGovernor = pendingGovernor` and then you don't need to define it below? and just reuse this for the require checks

Reply:
ah good catch.